software-mansion/react-native-screens
# Add reproduction to Issue #2125

## Description
This PR aims to reproduce and solve #2125. Following the discussion in [PR2175](https://github.com/software-mansion/react-native-screens/pull/2175#issuecomment-2379112183) I create a reproduction for the issue.


## Changes
- Create Test2125 with reproduction to issue #2125 





## Test code and steps to reproduce

Reproduction(Architecture: Paper, System: IOS) : 
1. Start the application and click "Go To Second Screen" button
2. Click "Home" button to navigate back - modal should appear.
3. Click "Cancel" button in the modal.
4. Sometimes App is frozen, if not, repeat step 2 and 3 many times.

Video: 


https://github.com/user-attachments/assets/c8e25721-375e-45f1-a3ed-aad092896cc9




## Checklist
